go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/prebid/prebid-server/v3
 
-go 1.21
+go 1.23
```

Comment:
How is the "supporting two releases" done by choosing an older version? Do you mean, instead, that you always stay on the "previous release"?

Reply:
Setting to 1.23 means that we can't start using the [iter](https://pkg.go.dev/iter) package yet.

Reply:
Here's an example file we can add to an iterators module in util

```go
package iterators

import "iter"

// SlicePointers returns an iterator that yields indices and pointers to the elements of a slice.
func SlicePointers[Slice ~[]T, T any](s Slice) iter.Seq2[int, *T] {
	return func(yield func(int, *T) bool) {
		for i := range s {
			if !yield(i, &s[i]) {
				break
			}
		}
	}
}

// SlicePointerValues returns an iterator that yields pointers to the elements of a slice.
func SlicePointerValues[Slice ~[]T, T any](s Slice) iter.Seq[*T] {
	return func(yield func(*T) bool) {
		for i := range s {
			if !yield(&s[i]) {
				break
			}
		}
	}
}
```

Reply:
> How is the "supporting two releases" done by choosing an older version? Do you mean, instead, that you always stay on the "previous release"?

We build the Docker image using the latest version to take advantage of compiler / runtime improvements and restrict language features to the previous version.

> Not setting to 1.23 means that we can't start using the [iter](https://pkg.go.dev/iter) package yet, which can make some utilities that improve walking, say… []Bid (by walking the pointers to the Bid rather than making a copy of them for each iteration.

Correct. We'll wait a little bit for those new features.